server/vulnerabilities/msrc/msrc_api.go

```diff
@@ -49,7 +49,7 @@ func feedName(date time.Time) string {
 }
 
 func (msrc MSRCClient) getURL(date time.Time) (*url.URL, error) {
-	return url.Parse(msrc.baseURL + "/cvrf/v2.0/document/" + feedName(date))
```

Comment:
curious if there's any notable differences we care about in the v3 feed.  Maybe we can grab a diff from the last successful release

Reply:
There's no difference in content for CVRF, and the v2 API has redirected to v3 for a few months now: https://msrc.microsoft.com/blog/2024/07/announcing-the-cvrf-api-3.0-upgrade/

Reply:
I also md5'd 2020-Jan and got the same thing between v2.0 and v3.0 (which I'd expect since v2.0 302s to v3.0)